test.md

```diff
@@ -45,12 +45,15 @@ These functions make assertions about the state of the `<stack>` cell.
 
 ```k
     syntax Assertion ::= "#assertTopStack" StackItem String
+                       | "#assertTopStackExactly" StackItem String
                        | "#assertStack"    Stack     String
  // -------------------------------------------------------
```

Comment:
Formatting here is kinda wonky, let's either line up all the arguments or none of them.